src/presets.asm

```diff
@@ -200,14 +200,13 @@ preset_load_preset:
 
     LDX #$0000
   .loop_path
-    LDA $C1 : STA $7FF000,X
+    LDA $C1 : STA $7F0002,X
```

Comment:
All my work to save the $7F bank, and now I see this... ugh.
Remember that row of boxes Ruya found?  This could be the reason... although Ruya typically uses custom presets that don't use this logic, so maybe not?  Maybe he started with regular presets in his practice session.

Using $7F0002,X should be fine since we haven't loaded level data yet, and when we do it will overwrite this.

Reply:
This change causes some of my presets to load OoB. Double checked the last_preset pointer and it is correct.

I also encountered one crash going through a door after the sound freaked out and played a constant annoying tone, but I think that might have been this change rather than the music changes. Haven't seen it since swapping back to $7FF000.

Reply:
I don't understand why changing $7FF000 to $7F0002 would break anything.  It's just your unpublished presets?  I wonder what the difference would be.

Reply:
Tested this again and it's not causing me any problems now. I think I had one of them set to $7FF002 by mistake. That seems to give similarly bad results as I was seeing this morning. Sorry for false alarm